tests/javascript/index.js

```diff
@@ -12,4 +12,8 @@ export class NI {
     makeAwesomer() {
         this._awesomeLevel += 1;
     }
+
+    async asyncAwesomeness() {
+        await new Promise();
```

Comment:
nit / doesnt really matter, but seeing a promise that will never resolve just feels weird
```suggestion
        await Promise.resolve(42);
```